portal/portal-render-impl/impl/src/java/org/sakaiproject/portal/render/portlet/services/SakaiOptionalPortletContainerServices.java

```diff
@@ -281,16 +277,16 @@ private void setupThread(PortletRequest request, boolean doLog)
 
 		// *** Testing Printout to see how well we have the APIs Configured ****
 		// ToolSession ts = SessionManager.getCurrentToolSession();
-		// System.out.println("*** TEST *** \nTool Session = "+ts);
+		// log.debug("*** TEST *** \nTool Session = {}", ts);
```

Comment:
I didn't want to mess with Chuck's logic. I thought I'd leave those for maybe Chuck to revisit and clean up.